lib/OpenQA/Scheduler/Model/Jobs.pm

```diff
@@ -458,7 +458,7 @@ sub _assign_multiple_jobs_to_worker ($self, $jobs, $worker, $directly_chained_jo
     if (my $tmpdir = $worker->get_property('WORKER_TMPDIR')) {
         File::Path::rmtree($tmpdir);
     }
-    my %worker_properties = (JOBTOKEN => random_string(), WORKER_TMPDIR => tempdir());
+    my %worker_properties = (JOBTOKEN => random_string(), WORKER_TMPDIR =>  tempdir(sprintf('webui.worker-%d.XXXXXXXX', $worker_id), TMPDIR => 1));
```

Comment:
This model is actually used by the scheduler. 😁

Reply:
ok, I kept it

Reply:
I would have preferred unique templates here, because then there would be no guessing which one was responsible if we ever see a related error in the logs again. Now it could be either. 😉

Reply:
Make a PR :)